addOns/ascanrules/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Leave data empty instead of adding "N/A" for the scan rules:
   - Cross Site Scripting (Persistent) - Prime
   - Cross Site Scripting (Persistent) - Spider
+- Maintenance changes.
```

Comment:
Re: the classes where just the test method was added. Worth mentioning?

Reply:
If the changes don't affect main code we don't need to say anything.